dpypelines/s3_tar_received.py

```diff
@@ -70,6 +71,21 @@ def start(s3_object_name: str):
         notifier.failure()
         raise err
 
+    #This method will use a schema to validate the manifest.json 
+    try:
+        schema_path = "schemas/validate_manifest.json"
+        validate_json_schema(schema_path=schema_path,data_dict=manifest_dict)
+        logger.info(
+            "Successfully validated manifest.json",
+            data={
+                "pattern_looked_for": "manifest.json",
```

Comment:
Change `"pattern_looked_for": "manifest.json"` to `"manifest_dict": manifest_dict`